pkg/client/object.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"fmt"
+	"sort"
+
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type CompareToolchainObjects func(firstObject, secondObject runtimeclient.Object) (bool, error)
```

Comment:
is this type still used anywhere? It doesn't seem so.

Reply:
> Nice cleanup 👍
> Shouldn't be also renamed all the related functions/types so they don't contain the removed `ToolchainObject` type name? The same applies also to the comments.

Actually, I did such renaming in sandbox-sre (PR coming soon), but I may have forgotten to do it in a few other places. Let me check that!

Reply:
(looks like my response to your comment landed at the wrong location 😮 )

Reply:
done in 573a24b